flake.lock

```diff
@@ -58,13 +58,54 @@
         "type": "github"
       }
     },
+    "nixify": {
+      "inputs": {
+        "crane": "crane",
+        "flake-utils": "flake-utils",
+        "nixlib": [
+          "nixlib"
+        ],
+        "nixpkgs": [
+          "nixpkgs"
+        ],
+        "rust-overlay": "rust-overlay"
+      },
+      "locked": {
+        "lastModified": 1665662838,
+        "narHash": "sha256-v/O5BaaHc1X0DgxqN9MmQ/96RLL2G0/Gr7Teiy6wWto=",
+        "owner": "rvolosatovs",
```

Comment:
How hard will it be/how long will it take to get off of your custom fork and begin using upstream?

Reply:
This is done now, btw, just need to update the dependency.
I'd rather do that in a follow-up